source/jquery.flot.navigate.js

```diff
@@ -129,8 +138,15 @@ can set the default in the options.
     var PANHINT_LENGTH_CONSTANT = $.plot.uiConstants.PANHINT_LENGTH_CONSTANT;
 
     function init(plot) {
+        plot.hooks.processOptions.push(initNevigation);
+    }
+
+    function initNevigation(plot, options) {
         var panAxes = null;
         var canDrag = false;
+        var panModes = options.pan.mode.split(' '),
```

Comment:
See comments above. This is not going to work when you have more appropriate values for 'mode'.

Reply:
Will be fixed in the following commits.